src/screens/Login/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { useRef, useState, useEffect } from 'react'
+import Card from '../../components/card/Card'
+import backgroundImage from '../../images/background_image.png'
+import Background from '../../components/background/Background'
+import Header from '../../components/title/header/Header'
+import InputField from '../../components/input/inputfield/InputField'
+import Button from '../../components/button/Button'
+import axiosInstance from '../../utils/api/apiservice'
+// Component handling the login functionality for the user
+const Login = () => {
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth)
+  const imageUrl = windowWidth >= 480 ? backgroundImage : 'none'
+
+  useEffect(() => {
+    const handleWindowResize = () => {
+      setWindowWidth(window.innerWidth)
+    }
+
+    window.addEventListener('resize', handleWindowResize)
+
+    return () => {
+      window.removeEventListener('resize', handleWindowResize)
+    }
+  }, [])
+  const userRef = useRef<HTMLInputElement>(null)
+
+  // TODO: these variables will be used for the API call to backend - to be uncommented then
+  const [user, setUser] = useState('')
+  const [password, setPassword] = useState('')
+
+  const handleLogin = async (event: any) => {
+    // axios api
+    event.preventDefault()
+    try {
+      const response = await axiosInstance.post('/user/login', {
+        email: user,
+        password: password,
+      })
+      console.log(response)
+    } catch (err) {
+      console.log(err)
+    }
+  }
+
+  return (
+    <Background classname='App' backgroundImage={imageUrl}>
+      <Card>
+        <Header classname='login_header' content='Asset Mappr' />
+        <Header classname='subheader' content='Login' />
```

Comment:
Should the tag be 'Subheader' component ?

Reply:
Component needs to be renamed. This will proceed as we move ahead. Will add a TODO.

Reply:
Sure